server/eventlog/eventlog.go

```diff
@@ -341,6 +350,9 @@ func (w *EventLogWriter) writeChunkToKeyValStore(ctx context.Context, writeReque
 		curChunk,
 	)
 	w.lastChunk = curChunk
+	if w.pubsub != nil {
+		w.pubsub.Publish(ctx, w.pubsubChannel, "")
```

Comment:
is my understanding right that this dangles the pubsub stream value as an empty string for 8 hours? if so, is there a way to just kill the stream here instead?

Reply:
We don't want to kill the stream here, since this is where the log is being updated - the empty string is just a signal for clients to poll the tail of the log. The empty string will dangle for 8h but I think it won't take up enough memory to be worth worrying about